README.md

```diff
@@ -107,6 +107,22 @@ let hcaptcha = try? HCaptcha(
 
 Note: this should be the **bare** host, i.e. not including a protocol prefix like https://.
 
+#### Change widget theme
+
+SDK supports two built-in themes: `light` and `dark`
```

Comment:
```suggestion
The SDK supports three built-in themes: `light`, `dark`, and `contrast`
```

Meta-comment: not sure adding indirection vs. simply passing a string is a good idea here, as more themes will be introduced in the future. The current design forces an SDK update for each one..

Reply:
@e271828- generally I have no objections against this.

BTW how SDK will behave if an unknown theme passes?
Do we need some error handling for this case or cover this case with Unit tests?

Reply:
Done

Reply:
The SDK will simply ignore a malformed theme name, and log an error to the console: `"Cannot find theme with name: <fake>"`

So, probably pretty safe.